Loop/Managers/LoopDataManager.swift

```diff
@@ -1907,7 +1996,7 @@ protocol LoopDataManagerDelegate: AnyObject {
     ///
     /// - Parameters:
     ///   - units: The recommended bolus in U
-    /// - Returns: a supported bolus volume in U. The volume returned should not be larger than the passed in rate.
+    /// - Returns: a supported bolus volume in U. The volume returned should be the nearest deliverable volume.
```

Comment:
[nit] should this comment how rounding is handled?

Reply:
Not 100% sure I understand what's being asked here. Are you suggesting that we should not comment on how it's implemented?